<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1411</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="containerView" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>-1</y>
      <width>921</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>560</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>560</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QSlider" name="volume_slider_">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>60</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="attachBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>attachPlayer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detachBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>500</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>detachPlayer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="publishAudioBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>500</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>publishAudio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unPublishAudio">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>500</y>
      <width>211</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>unpublishAudio</string>
    </property>
   </widget>
   <widget class="QSlider" name="seekSlider">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>720</y>
      <width>821</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="publishVideoBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>publishVideo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unpublishVideoBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>630</y>
      <width>201</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>unpublishVideo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputUrl">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>610</y>
      <width>911</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>http://compress.mv.letusmix.com/1c4dbe546537a9459d7b2c208d513303.mp4</string>
    </property>
    <property name="placeholderText">
     <string>Input your playing url</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>20</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>LocalVolume</string>
    </property>
   </widget>
   <widget class="QSlider" name="remote_volume">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>150</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>110</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>RemoteVolume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseBtn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>560</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>pause</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1411</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAgoraMediaPlayerDemo">
    <property name="title">
     <string>AgoraMediaPlayerDemo</string>
    </property>
   </widget>
   <addaction name="menuAgoraMediaPlayerDemo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
